ovh/resource_cloud_project_failover_ip_attach.go

```diff
@@ -210,9 +206,8 @@ func resourceCloudProjectFailoverIpAttachCreate(d *schema.ResourceData, meta int
 }
 
 func resourceCloudProjectFailoverIpAttachDelete(d *schema.ResourceData, meta interface{}) error {
-	// Failover IPs cannot be deleted, the best that can be done in this instance is to check it exists
-	if err := resourceCloudProjectFailoverIpAttachRead(d, meta); err != nil {
-		return err
-	}
+	// Failover IPs cannot be detached from an instance, so nothing done here.
+	d.SetId("")
+
```

Comment:
@amstuta  Does this fix https://github.com/ovh/terraform-provider-ovh/issues/780?

Reply:
@yellowhat no sorry, this is just a clean of the resource but it won't solve this issue. Regarding the problem you raised, the concerned team is checking why it happens and working on a fix.